### ISSUE TITLE
Notebook cell execution blocks on notification action when no `pip` is available

### ISSUE THREAD

**Issue by joyceerhl** [microsoft/vscode#174986]
Testing #174637

1. Create a codespace for microsoft/vscode, the kitchen sink image will not have pip in it
2. Try to run a Jupyter notebook cell 
3. Get an error notification saying that there's no pip
4. :bug: the kernel and cell are in a pending state until the notification is actioned. This is weird as an end user because when I see this error message, my first move is to go to the terminal to install the missing dependencies, and subsequently it wasn't clear what I needed to do to unblock the cell execution. IMO if a kernel dependency is missing and the only action available in the notification is to open the docs, the cell execution should fail immediately

**Comment by rebornix** [comment]
/duplicate of https://github.com/microsoft/vscode/issues/175044

**Comment by joyceerhl** [comment]
@rebornix it's possible that a user has a Linux machine which doesn't have `pip` so I'm not sure that this is a complete duplicate of  https://github.com/microsoft/vscode/issues/175044. This issue is purely about the way that Jupyter handles the notification about missing pip within the kernel execution.
